lib/Configuration.js

```diff
@@ -34,68 +37,11 @@ const DEFAULT_CONFIG = {
   useRelativePaths: true,
 };
 
-const ENVIRONMENT_CORE_MODULES = {
-  meteor: [
-    'meteor/accounts-base',
-    'meteor/blaze',
-    'meteor/check',
-    'meteor/ddp-client',
-    'meteor/ddp-rate-limiter',
-    'meteor/ejson',
-    'meteor/email',
-    'meteor/http',
-    'meteor/check',
-    'meteor/meteor',
-    'meteor/mongo',
-    'meteor/random',
-    'meteor/reactive-var',
-    'meteor/session',
-    'meteor/templating',
-    'meteor/tracker',
-  ],
-  // As listed in https://github.com/nodejs/node/tree/master/lib
-  //
-  // Note that we do not include `process` in this list because that is
-  // available globally in node environments and will cause the following error
-  // if imported:
-  //
-  //   Error: Cannot find module 'process' from 'Foo.js'
-  node: [
-    'assert',
-    'buffer',
-    'child_process',
-    'cluster',
-    'console',
-    'constants',
-    'crypto',
-    'dgram',
-    'dns',
-    'domain',
-    'events',
-    'fs',
-    'http',
-    'https',
-    'module',
-    'net',
-    'os',
-    'path',
-    'punycode',
-    'querystring',
-    'readline',
-    'repl',
-    'stream',
-    'string_decoder',
-    'sys',
-    'timers',
-    'tls',
-    'tty',
-    'url',
-    'util',
-    'v8',
-    'vm',
-    'zlib',
-  ],
-};
+const MERGABLE_CONFIG_OPTIONS = [
```

Comment:
At this point in the code, I don't really understand what "mergable" means. Maybe it is worth adding a comment?